src/components/Card/Card.jsx

```diff
@@ -0,0 +1,629 @@
+import { useState } from "react";
+import "../../styles.css";
+import "./card-styles.css";
+import { AddToWishListHeart, Dismiss } from "../../assets";
+
+export const Card = () => {
+  const [addToWishlist, setAddToWishlist] = useState(false);
+  return (
+    <div className="w-full m-auto mb-8 text-left">
+      <h1 className="mb-6 text-3xl">Card</h1>
+
+      <div className="flex justify-between items-start mb-6">
+        <div className="mr-12">
+          {/* e-commerce card */}
+          <div className="card">
+            <img
```

Comment:
Cards could use some border ... again depends on the design but I would prefer cards with some border (as light as #eee)